frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx

```diff
@@ -538,7 +538,7 @@ export const DEPRECATED_CreateOperandForm: React.FC<OperandFormProps> = ({
     const schemaFields = fieldsForOpenAPI(
       schema?.properties?.spec as JSONSchema6,
       providedAPI,
-      formData,
+      formData as K8sResourceKind,
```

Comment:
Don't cast to K8sResourKind, figure out what's wrong with the underlying type definition.